src/js/easymde.js

```diff
@@ -878,18 +878,10 @@ function afterImageUploaded(editor, url) {
     var cm = editor.codemirror;
     var stat = getState(cm);
     var options = editor.options;
+    // TODO: Get the image name from the original file name
```

Comment:
Is this TODO something that needs to be added to this PR?

Reply:
No. If we want to add this feature, it will be a breaking change for current API.